### DISCUSSION TITLE
Storybook server returns "SyntaxError: Unexpected token ':'" for index.json

### DISCUSSION THREAD

**Discussion by dtrenz** [storybookjs/storybook#21988] [Help]
I'm in the process of upgrading Storybook (React) from 6.5 to 7.0.2 but after migrating, when I run Storybook I see "SyntaxError: Unexpected token ':'".

It appears that this error is what is returned from `/index.json`, so it looks like something is causing an error during story indexing, maybe?

<img width="1046" alt="image" src="https://user-images.githubusercontent.com/639833/230475963-dc5776e7-d80c-448f-a7b0-5e0c40575d51.png">

 **Any ideas or tips on how to further troubleshoot the issue?** 

I set the `stories` config property to an empty array and I still see the same error, which makes me think that it has nothing to do with the actual stories files.

Any help is greatly appreciated. Thanks

**Comment by etlovett** [comment]
@dtrenz I see that you closed this. Were you able to figure out a resolution? I'm upgrading a project to Storybook v7.x and running into this exact same issue. If you have any tips for how to resolve this, it'd be much appreciated!

**Comment by dtrenz** [comment]
@etlovett Hmmm... I was able to upgrade, but I can't recall how I specifically got past this issue.

Maybe it was changing the storybook start script in my `package.json` from this...

```
"storybook": "start-storybook --ci -p 6006",
```

...to this?

```
"storybook": "storybook dev --ci -p 6006",
```

Sorry I can't be more helpful. Good luck!
